testbot.js

```diff
@@ -773,6 +789,11 @@ async function handleSellBuyCommand(msg, args) {
     // Calculate final price based on market price and modifier
     finalPrice = calculateFinalPrice(priceInfo.price, resolvedPriceModifier);
 
+    if (validatePriceModifier(priceModifier, finalPrice) == false) {

```

Comment:
Easy to understand error highlighting what the user is doing wrong with the command